fiftyone/zoo/models/torch.py

```diff
@@ -88,6 +93,145 @@ def _load_network(self, config):
         return model
 
 
+class TorchCLIPModelConfig(
```

Comment:
Per my above suggestion, I think it will be cleaner to move this to `fiftyone/utils/clip`, since this implementation is specific to CLIP.

`TorchvisionImageModelConfig` is here because it is a generic Torch construct (any torchvision model).

Reply:
makes sense, moved CLIP and its config to `utils/clip` subpackage